scripts/qa_crowbarsetup.sh

```diff
@@ -3170,6 +3170,11 @@ function oncontroller_run_tempest
     fi
     testr last --subunit | subunit-1to2 > tempest.subunit.log
 
+    # output the slowest tests from the latest run
```

Comment:
maybe this should only be executed when $tempestret eq to 0 (see if condition just above)?

Reply:
Done